src/main/java/hudson/plugins/performance/parsers/ParserDetector.java

```diff
@@ -112,6 +115,17 @@ private static boolean isJMeterSummarizerFileType(String firstLine, final Buffer
     }
 
     /**
+     * Detect LoadRunner MDB file using MS Access magic pattern "....Standard Jet DB"
+     * http://www.garykessler.net/library/file_sigs.html
+     */
+    private static boolean isLoadRunnerFileType(String line) throws IOException {
```

Comment:
Can we make this method more simple and understandable? 
maybe: 
```
 String patternString = "Standard Jet DB";
 return line.length() > patternString.length() &&
                patternString.equals(line.substring(4, line.length()));
```

Reply:
Sure, no problem.
Just want to make clear that it's a binary file format.
How about this...